hack/build/docker/cdi-func-test-file-host-init/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM fedora:28
 
 RUN mkdir -p /tmp/shared /tmp/source
 
-RUN yum install -y qemu-img
+RUN yum install -y qemu-img && dnf clean all
```

Comment:
Consider converting to use "dnf install" and being consistent?  Also, the Fedora images include the base repos so this works but do we want to do a "dnf update -y" before the install?  Many of the OS images won't work at all without it.

Reply:
yeah, I was amazed that this stuff actually installed without the update.  But it does.  Seeing as none of the other images do an update, I'll keep as-is for now.  But maybe I'll open an issue to investigate.  The reason I added the clean was to make the image smaller.  `make cluster-sync` takes awhile and every bit helps.